dotnet/CONTRIBUTING.md

```diff
@@ -14,3 +14,9 @@ Alternatively, run `msbuild` from this directory.
 
 Keep in mind that this will only run unit tests. The acceptance tests are only
 run when you build with `make`.
+
+## Make a release
+
+
+    # Change version in `Gherkin.NuGetPackages\Gherkin.nuspec`
+    .nuget\NuGet.exe pack Gherkin.NuGetPackages\Gherkin.nuspec
```

Comment:
The command

    mono --runtime=v4.0 .nuget/NuGet.exe pack Gherkin.NuGetPackages/Gherkin.nuspec

fails with the following error:

```
 Attempting to build package from 'Gherkin.nuspec'.
File not found: 'bin\Gherkin.dll'
```

This is not a path separator issue - there is no `Gherkin.dll` file in `bin`.

I could get it to work by changing `bin\Gherkin.dll` to `bin/Gherkin.dll` in the `.nuspec` file and then run this:
```
cd Gherkin.NuGetPackages
mono --runtime=v4.0 ../.nuget/NuGet.exe pack Gherkin.NuGetPackages/Gherkin.nuspec
```

I'm not sure if this is how you intended it to be. Is it ok to use forward slash for the source in the `.nuspec` file? If not, could we leave the dll in the same directory as the `.nuspec` file to avoid path issues?